docs/writing-docs/build-documentation.mdx

```diff
@@ -33,7 +33,7 @@ There's some caveats to this build mode, as to the normal Storybook build:
 
 ## Publish Storybook's documentation
 
-You can also publish your documentation, the same you would [publish](../sharing/publish-storybook.mdx) your Storybook. You can use the `--docs` flag with `build-storybook` command. We recommend as well including it as a script in your `package.json` file:
+You can also publish your documentation, the same you would [publish](../sharing/publish-storybook.mdx) your Storybook. You can use the `--docs` flag with `storybook build` command. We recommend as well including it as a script in your `package.json` file:
```

Comment:
Left a minor suggestion for you to look into when you can. Once again, adding the link to the available options for the `build` command, so that the user is aware of this and the other ones as well.
```suggestion
You can also publish your documentation the same you would [publish](../sharing/publish-storybook.mdx) your Storybook. You can use the `--docs` flag with the [`storybook build`](../api/cli-options.mdx#build) command. We recommend as well including it as a script in your `package.json` file:
```